src/programs/tests/JParameterManagerTests.cc

```diff
@@ -195,6 +195,7 @@ TEST_CASE("JParameterManager_VectorParams") {
         std::vector<std::string> outputs;
         auto param = jpm.GetParameter("test", outputs);
         REQUIRE(param->GetValue() == "first,second one, third one ");
+        REQUIRE(inputs.size()==3); // an additional test to see that the size of the input vector remains the same: Issue #256
```

Comment:
Nicely done! I like that the regression test is minimal and links back to the issue.